docs/pages/versions/unversioned/sdk/battery.md

```diff
@@ -30,10 +30,12 @@ export default class App extends React.Component {
     batteryLevel: null,
   };
 
-  async componentDidMount() {
+  componentDidMount() {
+    (async () => {
     let batteryLevel = await Battery.getBatteryLevelAsync();
     this.setState({ batteryLevel });
     this._subscribe();
+    })();
   }
```

Comment:
let's match the sdk37 docs and use:
```
  async componentDidMount() {
    let batteryLevel = await Battery.getBatteryLevelAsync();
    this.setState({ batteryLevel });
    this._subscribe();
  }
```

Reply:
Changing it to async gives another warning that `setState` should not be called in componentDidMount

![image](https://user-images.githubusercontent.com/6184593/82215329-8dc4bd80-9917-11ea-97c5-83d6188e98fd.png)